makefiles/libpcap.mk

```diff
@@ -0,0 +1,54 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS   += libpcap
+LIBPCAP_VERSION := 1.10.1
+DEB_LIBPCAP_V   ?= $(LIBPCAP_VERSION)
+
+libpcap-setup: setup
+	$(call GITHUB_ARCHIVE,the-tcpdump-group,libpcap,$(LIBPCAP_VERSION),libpcap-$(LIBPCAP_VERSION))
+	$(call EXTRACT_TAR,libpcap-$(LIBPCAP_VERSION).tar.gz,libpcap-libpcap-$(LIBPCAP_VERSION),libpcap)
+	mkdir -p $(BUILD_WORK)/libpcap/build
```

Comment:
Is a separate folder for building necessary?

Reply:
Well, every other cmake project does it, so I consider it a standard

Reply:
fair enough.